### PR TITLE
feat: added common types

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
-After cloning this template, please do the following:
-1. insert secrets to repo secrets for the github actions.
-2. replace every string "ts-npm-package-boilerplate" with your package name.
+this package provides common Map-Colonies types
+
+---
+## Installation
+
+``` 
+$ npm install @turf/turf
```
Comment (asafMasa) [inline README.md:9]: it will be installed via npm i, no need to be specified here

**Review by asafMasa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (asafMasa) [reply README.md:9]: @MapColonies/types

File: src/enums/geo/epsg.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum EPSG {
```
Comment (CL-SHLOMIKONCHA) [inline src/enums/geo/epsg.ts:1]: should talk with @syncush about it

Comment (alebinson) [reply src/enums/geo/epsg.ts:1]: Think this one should be proxied (as an example for type proxy principles)
`import { epsg } from 'EPSG_LIST_PKG';`

`export type Projections = epsg;`

`export const ExportProjections = {`
`  "EPSG:4326": epsg["EPSG:4326"],`
`  "EPSG:3857": epsg["EPSG:3857"],`
`  ...`
`};`

**Review by alebinson — CHANGES_REQUESTED** [review]
See my comment

File: src/enums/geo/epsg.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/enums/geo/epsg.ts]: merged your branch with the fix

File: src/enums/geo/GENERATED/epsg_codes_type.ts [hunk 3]
```diff
@@ -0,0 +1,8113 @@
+export type EpsgCode =
```
Comment (syncush) [inline src/enums/geo/GENERATED/epsg_codes_type.ts:1]: formatting issue with this file

File: src/enums/geo/GENERATED/epsg_interface.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable naming-convention  */
```
Comment (syncush) [inline src/enums/geo/GENERATED/epsg_interface.ts:1]: why?

File: tests/configurations/unit/jest.config.js [hunk 5]
```diff
@@ -16,11 +16,12 @@ module.exports = {
     ['jest-html-reporters', { multipleReportsUnitePath: './reports', pageTitle: 'unit', publicPath: './reports', filename: 'unit.html' }],
```
Comment (syncush) [inline tests/configurations/unit/jest.config.js:16]: you should test your code, you have scripts generating interfaces. I do not accept it

**Review by syncush — CHANGES_REQUESTED** [review]

File: src/enums/geo/GENERATED/epsg_interface.ts (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply src/enums/geo/GENERATED/epsg_interface.ts]: removed - ignored generated file

File: src/enums/geo/GENERATED/epsg_codes_type.ts (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply src/enums/geo/GENERATED/epsg_codes_type.ts:1]: ignored generated file

**Pull request merged**